spec/src/main/asciidoc/ch09-container-provider-contracts.adoc

```diff
@@ -20,9 +20,7 @@ discovering the _persistence.xml_ files and processing them.
 When the container finds a _persistence.xml_
 file, it must process the persistence unit definitions that it contains.
 The container must validate the _persistence.xml_ file against the
-_persistence_2_2.xsd_, _persistence_2_1.xsd_, _persistence_2_0.xsd_,
-or _persistence_1_0.xsd_ schema in accordance with the version specified
-by the _persistence.xml_ file and report any validation errors. Provider
+_persistence_3_0.xsd_ schema and report any validation errors. Provider
```

Comment:
I think that persistence_2_2.xsd can also be mentioned as well as per https://eclipse-ee4j.github.io/jakartaee-platform/minutes/2020-01-28.html:

> 3. Support for Jakarta EE 8 but not earlier schemas is required in Jakarta EE 9 products; support for schemas earlier than Jakarta EE 8 is optional in Jakarta EE 9 products. Votes: 7

Reply:
Addressed.